kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/write/storageApi/StorageWriteApiDefaultStreamTest.java

```diff
@@ -0,0 +1,271 @@
+package com.wepay.kafka.connect.bigquery.write.storageApi;
+
+import com.google.api.core.ApiFuture;
+import com.google.cloud.bigquery.storage.v1.JsonStreamWriter;
+import com.google.cloud.bigquery.storage.v1.AppendRowsResponse;
+import com.google.cloud.bigquery.storage.v1.RowError;
+import com.google.cloud.bigquery.storage.v1.TableName;
+import com.google.cloud.bigquery.storage.v1.Exceptions;
+
+import com.google.rpc.Status;
+import com.wepay.kafka.connect.bigquery.exception.BigQueryStorageWriteApiConnectException;
+import io.grpc.StatusRuntimeException;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.json.JSONObject;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.ArgumentMatchers;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.CALLS_REAL_METHODS;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.doNothing;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+public class StorageWriteApiDefaultStreamTest {
+
+    TableName mockedTableName = TableName.of("dummyProject", "dummyDataset", "dummyTable");
+
+    @Test
+    public void testDefaultStreamNoExceptions() throws Exception {
```

Comment:
I see lot of similarity in test code of Exception and Error testcase for same type, for example - `testDefaultStreamMalformedRequestError` and `testDefaultStreamMalformedRequestException`

Would it be possible to avoid this duplication by combining these testcase or writing some helper function?

Reply:
These tests target different places and should be kept separate. I have now moved most of the logic to helper function to avoid duplication of common pieces